datacube/index/abstract.py

```diff
@@ -1141,30 +1141,32 @@ def archive(self, ids: Iterable[DSID]) -> None:
         :param Iterable[Union[str,UUID]] ids: list of dataset ids to archive
         """
 
-    def archive_less_mature(self, ds: Dataset) -> None:
+    def archive_less_mature(self, ds: Dataset, delta: int) -> None:
```

Comment:
Seems like this is missing the default value added in my latest PR

Reply:
D'oh!  Yes, I can see how that happened.  Thanks.